<launch>
  <node pkg="rgbd_calibration" 		    	exec="get_calibration_images_node" name="get_calibration_images_node" output="screen">
		<param name="image_topic" 	  	 	value="/camera/ir/image" />
		<param name="checkerboard_rows" 	value="6" />
		<param name="checkerboard_cols" 	value="9" />
		<param name="folder_path" 	      	value="/media/ehambram/Data1/Datasets/xtion_calibration/test" />
		<param name="capture_frequency" 	value="0.2" />
	</node>
</launch>